.travis/test_runner.sh

```diff
@@ -10,13 +10,15 @@
 
 set -e
 
-TESTS="$1"
+TESTS="$@"
 COMMAND="coverage run manage.py test --noinput --failfast --traceback --verbosity=2"
 
 /moto-s3/env/bin/moto_server s3 &
 
 if [ -z ${COMMAND_OVERRIDE} ]; then
+    echo "Running tests: $TESTS"
     $COMMAND $TESTS
```

Comment:
sorry @millerdev I probably shouldn't have merged without context. feel free to revert if you want